public/app/features/alerting/unified/api/alertmanager.ts

```diff
@@ -51,6 +53,20 @@ export async function fetchAlertManagerConfig(alertManagerSourceName: string): P
   }
 }
 
+//this is only available for the "grafana" alert manager
+export async function fetchValidAlertManagerConfig(): Promise<AlertManagerCortexConfig[]> {
```

Comment:
How about trying to use RTK Query instead? There is already `public/app/features/alerting/unified/api/alertmanagerApi.ts` file which contains a few RTK Query endpoints for Alertmanager